CIP-XXXX/README.md

```diff
@@ -0,0 +1,447 @@
+
+#### 4. Nonce
+
+As with CIP-36 Voter Registration method, we must include a piece of data that changes to avoid a replay attack. As with
+CIP-36, and stake pool KES counter rotation, the nonce is expected to be an integer value that is incremented as new or
+updated registrations occur. The suggestion to use the current slot tip of the network at the time of submission from 
+CIP-36 is also recommended here.
+
+Validators and indexers observing token registrations should always honor the verified registration with the highest
+nonce value.
+
+**Example:**
+
+`12345`
+
+### Optional Fields
+
+The following are optional fields that may be included in the token registration to provide additional information or
+context to the registration.
+
+#### 5. Data Oracle URI
+
+To be utilized and expanded upon in a separate CIP, this should be a valid URI pointing to a source of additional,
+potentially dynamic information relating to the project and/or the tokens minted under it.
+
+**Example:**
+
+`'https://oracle.mytokenproject.io/'`
+
+#### 6. CIP-Specific Information
+
+This entry, if present, should be a CIP ID indexed object containing additional information pertaining to that CIP.
+When and where possible the CIP-Specific registration should follow the CBOR-like declaration syntax to ensure that
+the content is well-formed and easily parseable.
+
+These additional CIPs/Schemas to be determined by the community could include:
+
+- CIP-25/68 NFT Project top-level project information
+- CIP-26 FT Project monetary policy information
+- CIP-27 NFT Project Royalty information
+
+##### CIP-25 & CIP-68: Non-Fungible Tokens / Policy Information
+
+See [CIP25](CIP-25) for a description of a non-fungible token project policy registration. 
+([CIP25.json](CIP-25/CIP25.json) as an example, [CIP25.schema.json](CIP-25/CIP25.schema.json) for schema documentation).
+
+Because the information registered here is mostly helpful to aggregator services and marketplaces, it applies equally to
+both CIP-25 and CIP-68 metadata standards. A project utilizing one or the other should reference this documentation and
+include the relevant information under index #6, prefixed by the number of the CIP (25 or 68) depending upon the metadata
+format.
+
+##### CIP-26: Fungible Tokens / Monetary Policy
+
+See [CIP26](CIP-26) for a description of a fungible token specific registration ([CIP26.json](CIP-26/CIP26.json) as an 
+example, [CIP26.schema.json](CIP-26/CIP26.schema.json) for schema documentation). This information can replace the information
+currently housed in the [Cardano Token Registry](https://github.com/cardano-foundation/cardano-token-registry) and is 
+based on the format currently used in those registrations along with a few additional fields.
+
+##### CIP-27: Cardano NFT Royalty Information
+
+##### Inline Datum / Reference UTXO Registration
+
+Depending on the needs of a specific project or CIP, the CIP-Specific registration should include a reference pointer to 
```

Comment:
What is the format for the pointer? From the example below, it seems to be an address from the outputs and.. a hash of some sort?

Wouldn't it be sufficient to simply store a hash of the inline datum as a pointer? Application digesting this will have to crawl through the list of outputs anyway and are very much likely capable of calculating hash digests.

Reply:
This was added as a preliminary thought of a way to point explorers in the right direction of an on-chain inline datum. Just thinking it through a bit more, it would probably be better to use something like a "beacon token" that could contain inline datum data and so the identifier of that beacon token could be contained within CIP-88 registration and subsequently used by explorers or SCs that need the data housed there.

Reply:
I have revised this to reference a token that would be used to "pointer" to an inline datum txn (currently only present in the CIP-26 specific content). This way the inline datum on the txn hosting the reference token could be updated w/o issuing an updated policy registration.

Reply:
resolving since fixed in earlier revision.